integration_test/third_party_apps_data/applications/postgresql/expected_metrics.yaml

```diff
@@ -0,0 +1,25 @@
+- type: workload.googleapis.com/postgresql.backends
+  value_type: INT64
+  kind: GAUGE
+  monitored_resource: gce_instance
+  labels:
+    database: .*
+  representative: true
+- type: workload.googleapis.com/postgresql.commits
+  value_type: INT64
+  kind: CUMULATIVE
+  monitored_resource: gce_instance
+  labels:
+    database: .*
+- type: workload.googleapis.com/postgresql.db_size
+  value_type: INT64
+  kind: GAUGE
+  monitored_resource: gce_instance
+  labels:
+    database: .*
+- type: workload.googleapis.com/postgresql.rollbacks
+  value_type: INT64
+  kind: CUMULATIVE
+  monitored_resource: gce_instance
+  labels:
```

Comment:
I think postgresql has 7 metrics. 3 missing one shall mark as optional?

Reply:
We only document 4 for Ops Agent: https://cloud.google.com/monitoring/api/metrics_opsagent#opsagent-postgresql

Although the legacy agent has 7: https://cloud.google.com/monitoring/api/metrics_agent#agent-postgresql

The missing ones are:
* blocks_read_count (CUMULATIVE, INT64)
* num_tuples (GAUGE, DOUBLE)
* operation_count (CUMULATIVE, INT64)

We should try to see first why they're missing.

Reply:
Don't need to block the PR on this -- will investigate and open a follow-up PR if needed.